flyway-community-db-support/flyway-database-yugabytedb/src/main/java/org/flywaydb/community/database/postgresql/yugabytedb/YugabyteDBDatabase.java

```diff
@@ -30,6 +33,13 @@ public YugabyteDBDatabase(Configuration configuration, JdbcConnectionFactory jdb
 
     @Override
     protected YugabyteDBConnection doGetConnection(Connection connection) {
+        Statement stmt = null;
+        try {
+            stmt = connection.createStatement();
+            stmt.execute("set yb_silence_advisory_locks_not_supported_error=on;");
+        } catch (SQLException throwables) {
+            throwables.printStackTrace();
```

Comment:
We should use a logger from the logging framework and log at appropriate level (may be at `WARN` or `ERROR`). See `PostgreSQLDatabaseType`